### PR TITLE
feat(pm): add environment、cluster、project、app serrvice

### PR DESCRIPTION
**Pull request by codding-y**

### CODE REVIEW

File: src/BasicAbility/Masa.BuildingBlocks.BasicAbility.Pm/Model/AppModel.cs [hunk 1]
```diff
@@ -1,21 +1,24 @@
-namespace Masa.BuildingBlocks.BasicAbility.Pm.Model
+namespace Masa.BuildingBlocks.BasicAbility.Pm.Model;
+
+public class AppModel
```
Comment (doddgu) [inline src/BasicAbility/Masa.BuildingBlocks.BasicAbility.Pm/Model/AppModel.cs:3]: Model? Dto?

File: src/BasicAbility/Masa.BuildingBlocks.BasicAbility.Pm/Service/IProjectService.cs [hunk 2]
```diff
@@ -2,6 +2,14 @@ namespace Masa.BuildingBlocks.BasicAbility.Pm.Service
 {
     public interface IProjectService
     {
-        Task<List<ProjectModel>> GetProjectListAsync(string envName);
+        Task<List<ProjectAppsModel>> GetProjectAppsAsync(string envName);
+
+        Task<List<ProjectModel>> GetListByEnvironmentClusterIdAsync(int envClusterId);
```
Comment (doddgu) [inline src/BasicAbility/Masa.BuildingBlocks.BasicAbility.Pm/Service/IProjectService.cs:7]: envClusterId? or envId and clusterId?

Comment (codding-y) [reply src/BasicAbility/Masa.BuildingBlocks.BasicAbility.Pm/Service/IProjectService.cs:7]: envClusterId

**Review by doddgu — APPROVED** [review]
LGTM

**Pull request merged**